src/mcp/index.ts

```diff
@@ -39,6 +39,15 @@ export class FirebaseMcpServer {
     this.server.registerCapabilities({ tools: { listChanged: true } });
     this.server.setRequestHandler(ListToolsRequestSchema, this.mcpListTools.bind(this));
     this.server.setRequestHandler(CallToolRequestSchema, this.mcpCallTool.bind(this));
+    this.server.oninitialized = () => {
+      const clientInfo = this.server.getClientVersion();
+      if (clientInfo?.name) {
+        trackGA4("mcp_client_connected", {
```

Comment:
Do we want to include the client version information in `mcp_tool_call` metric below as well?

Reply:
Done!